src/main/java/org/terasology/metalrenegades/world/dynamic/MRBiome.java

```diff
@@ -49,19 +49,33 @@
         grass = blockManager.getBlock("CoreAssets:Grass");
         snow = blockManager.getBlock("CoreAssets:Snow");
         dirt = blockManager.getBlock("CoreAssets:Dirt");
+        Block sandstone = blockManager.getBlock("GenericRocks:Sandstone");
+        // Sandstone is the substrate that the other strata are in
+        // The most efficient way to achieve that is to just enter it multiple times in the array
```

Comment:
:+1: yep, that's a pattern we use in several places, either as explicit as here, or behind some abstraction layer.